playbook/bounties.md

```diff
@@ -0,0 +1,176 @@
+  * for coding bounties, the Worker is expected to push commits in WIP Pull Request.
+  * for writeups, the Worker is expected to share a draft Hackmd document (or other meidum of his choice) he is working at.
+* request a review once the work is done. Review is requested on Slack in a Circle's channel like this:
+  > 🌴👀 Requesting review for **\<bounty title\>** \
+  > Issue: \<link to the Bounty on Github\> \
+  > Product: \<link to deliverable, e.g. PR or writeup\>
+
+### Reviewer
+
+Reviewer is the person who reviews the Bounty's deliverables.
+
+Reviewer is expected to:
+
+* do review in timely manner, don't linger (see "Challenging bounties" section below)
+* ensure all the delivables are provided
+* ensure all the Scope items are addressed
+* for coding bounties:
+  * make sure the code is compiling/building correctly
+  * review the code for logical correctness, inefficiencies and style (TBD)
+  * ensure unit tests are passing
+  * ensure code coverage is not reduced
+  * ensure integration tests are passing for the PR branch
+* for public writeups
+  * do fact check
+  * check readability
+  * correct grammar and punctuation
+  * check social network preview e.g. twitter card (if applicable)
+* forward all the issues found publicly to the Worker (e.g. as comments on Github)
+* approve the PR once the review is complete successfuly (if applicable)
+
+## Challenging bounties
+
+Gardener/Worker/Reviewer roles may be taken over by other people in a certain situations (see below). Each takeover must be stated in written form in corresponding Github issue including new assignee and the reason for takeover.
+
+### Challenging worker and reviewer
+If worker or reviewer are not publishing any updates for the bounty for 2 working days, then anyone can challenge him and take over the worker role. Updates for worker role can be in any form, most notably: commits to WIP Pull Request, updates on product artifact (hackmd, presentation, diagram etc), review comments, updates in form of github comments or slack conversations.
+
+### Challenging gardener
+If gardener is not helping worker to find a reviewer or doesn't address worker's questions within 2 days, then anyone can challenge him and take over a gardener role on the bounty.
+
+## Payout
+
+Funds allocated for bounty are split between Gardener, Worker and Reviewer in a proportion defined by Gardener (can be negotiated).
+
+It is Gardener's duty to request the payout once the bounty is completed and reviewed. Payout is requested on Slack in a Circle's channel like this:
+  > 🌴💰 Requesting payout for **\<bounty title\>** \
+  > Issue: \<link to the Bounty on Github\> \
+  > Product: \<link to deliverable, e.g. PR or writeup\> \
+  > gardener: \<@gardener\> \<gardener share\> \
+  > worker: \<@worker\> \<worker share\> \
+  > reviewer: \<@reviewer\> \<reviewer share\>
+
+## Links
+
+* [Bounty issue template](https://github.com/leapdao/leap-node/blob/master/.github/ISSUE_TEMPLATE/bounty.md)
```

Comment:
```suggestion
* [Bounty issue template](https://github.com/leapdao/leap-node/blob/master/.github/ISSUE_TEMPLATE.md)
```

Reply:
that was on purpose :) Github allows many different issue templates — in this case we are creating the one for "Bounty"

Example:
![image](https://user-images.githubusercontent.com/163447/77681625-ae6f3780-6fa6-11ea-9cd3-f097e1e2951c.png)

Reply:
again.. the URL is such on purpose :)

Reply:
ok, does it link to `leap-node` repo on purpose? because there is no such file in that repo, and no open PR to add it.